src/openforms/forms/forms/form_statistics.py

```diff
@@ -50,3 +52,12 @@ class ExportStatisticsForm(forms.Form):
             "multiple options."
         ),
     )
+
+    def export(self) -> Dataset:
+        start_date: date = self.cleaned_data["start_date"]
+        end_date: date = self.cleaned_data["end_date"]
+        return export_registration_statistics(
```

Comment:
Maybe a bit picky, but why do you define the dates as variables? Is it (only) to specify the date typing?
Otherwise, why not just pass the `cleaned_data` directly?

Reply:
A couple of reasons, all summarized as "loose coupling".

1. A simple `export_registration_statistics` with plain parameters is easy to understand when you're in that function, without adding the cognitive overhead of knowing what the form structure looks like and what's in `cleaned_data`.
2. Building on top of 1., I can modify my form implementation and interface without needing to touch the actual export function. If I have to touch the export function, I'm very well aware that I'm changing a Python API and need to check for possible other call sites.
3. The export function can also be called through other user interfaces, e.g. a management command/CLI where the date parameters get passed/parsed differently, or a celery task could be wired up to periodically generate these and automatically prepare/send the exports.
4. Explicit type annotations are nice indeed :)